<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="getone">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>一抽</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getten">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>十连</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cardpool">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>2.5th Anniversary卡池</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="askbutton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>670</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>公告</string>
   </property>
   <property name="description">
    <string>更多详情请查阅公告</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>179</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>抽卡统计：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cardpool_Box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>2.5th Anniversary卡池</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>2.5th Anniversary卡池</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小林历奇&amp;北港火山 UP卡池</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北部玄驹&amp;里见光钻 UP卡池</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="change_cardpool">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认更换</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_getnum">
      <property name="text">
       <string>
★抽取次数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="value_getnum">
      <property name="text">
       <string>
0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ssrnum">
      <property name="text">
       <string>★SSR次数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="value_ssrnum">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_srnum">
      <property name="text">
       <string>★SR次数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="value_srnum">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_rnum">
      <property name="text">
       <string>★R次数：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="value_rnum">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Pssr">
      <property name="text">
       <string>★SSR出货率：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="value_Pssr">
      <property name="text">
       <string>0.00%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="view_pool">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="view_pool_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="getten_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>十连</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getone_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>一抽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cardpool_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>2.5th Anniversary支援卡池</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>170</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_getnum_3">
      <property name="text">
       <string>
★抽取次数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="value_getnum_3">
      <property name="text">
       <string>
0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ssrnum_3">
      <property name="text">
       <string>★SSR次数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="value_ssrnum_3">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_srnum_2">
      <property name="text">
       <string>★SR次数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="value_srnum_2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_rnum_3">
      <property name="text">
       <string>★R次数：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="value_rnum_3">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Pssr_3">
      <property name="text">
       <string>★SSR出货率：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="value_Pssr_3">
      <property name="text">
       <string>0.00%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_result_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>120</y>
     <width>179</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>抽卡统计：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_cardpool_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认更换</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cardpool_Box_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>590</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>2.5th Anniversary支援卡池</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>2.5th Anniversary支援卡池</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SSR目白高峰、丸善斯基 UP卡池</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SSR北部玄驹、神鹰UP卡池</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>211</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>211</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>getone</zorder>
  <zorder>getten</zorder>
  <zorder>label_cardpool</zorder>
  <zorder>askbutton</zorder>
  <zorder>label_result</zorder>
  <zorder>cardpool_Box</zorder>
  <zorder>change_cardpool</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>view_pool</zorder>
  <zorder>view_pool_2</zorder>
  <zorder>getten_2</zorder>
  <zorder>getone_2</zorder>
  <zorder>label_cardpool_2</zorder>
  <zorder>formLayoutWidget_2</zorder>
  <zorder>label_result_2</zorder>
  <zorder>change_cardpool_2</zorder>
  <zorder>cardpool_Box_2</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
